src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -672,7 +672,11 @@ export class AccessibleView extends Disposable {
 		const accessibleViewHelpProvider: IAccessibleContentProvider = {
 			id: lastProvider.id,
 			provideContent: () => lastProvider.options.customHelp ? lastProvider?.options.customHelp() : this._getAccessibleViewHelpDialogContent(this._goToSymbolsSupported()),
-			onClose: () => this.show(lastProvider),
+			onClose: () => {
+				this._contextViewService.hideContextView();
+				// HACK: Delay to allow the context view to hide #207638
+				setTimeout(() => this.show(lastProvider), 100);
```

Comment:
Does `setTimeout(, 0)` work? Otherwise there's always the possibility for it to be flaky. If not can you check if the operation is done? Should hideContextView return a promise?

Reply:
You may also be able to `queueMicrotask` as well which is lighter weight

Reply:
`setTimeout(,)` doesn't work. 

this was also used in these places and I haven't had any reports of flakyness

https://github.com/microsoft/vscode/blob/61935116e2b36b9df9c6dbbe288f0eb911e08c24/src/vs/workbench/contrib/accessibility/browser/accessibleView.ts#L683-L685
https://github.com/microsoft/vscode/blob/61935116e2b36b9df9c6dbbe288f0eb911e08c24/src/vs/workbench/contrib/accessibility/browser/accessibleView.ts#L323-L326

Reply:
i'll try `queueMicrotask`, I think I tried that awhile ago when @Tyriar had this same feedback for the above issues

Reply:
doesn't look `async` https://github.com/microsoft/vscode/blob/61935116e2b36b9df9c6dbbe288f0eb911e08c24/src/vs/base/browser/ui/contextview/contextview.ts#L359-L371

Reply:
setTimeout with a time of undefined/0 should be firing after the microtask, so maybe it didn't work or you got lucky or something 🤔

Reply:
yes that works